### PR TITLE
[Docs]: add Chinese version of config doc

### PR DESCRIPTION
**Pull request by BIGWangYuDong**
This PR adds a Chinese version of the config tutorial.

### CODE REVIEW

File: docs_zh-CN/tutorials/config.md [hunk 1]
```diff
@@ -1 +1,525 @@
-# 教程 1: 配置文件
+# 教程 1: 学习配置文件
+
+我们整合了模块化和继承设计融入到了我们的配置文件中，这便于进行各种实验。如果需要检查配置文件，可以通过运行 `python tools/misc/print_config.py /PATH/TO/CONFIG`  来查看完整的配置
```
Comment (hhaAndroid) [inline docs_zh-CN/tutorials/config.md:3]: 我们整合了模块化和继承设计融入到了我们的配置文件中？ 不太通顺

File: docs_zh-CN/tutorials/config.md [hunk 2]
```diff
@@ -1 +1,525 @@
...
+
+  一些配置字典在配置文件中以列表的形式存在。例如，数据训练流程 `data.train.pipeline` 通常是一个列表，比如 `[dict(type='LoadImageFromFile'), ...]`。如果需要将 `'LoadImageFromFile'` 改成 `'LoadImageFromWebcam'`，需要写成下述形式： `--cfg-options data.train.pipeline.0.type=LoadImageFromWebcam`。
+
+- 更新列表或元组的值
+
+  如果要更新的值是列表或元组。例如，配置文件通常设置 `workflow=[('train', 1)]`，如果需要改变这个键，可以通过 `--cfg-options workflow="[(train,1),(val,1)]"` 来重新设置。需要注意，引号 \" 是支持列表或元组数据类型所必需的，并且在指定值的引号内**不允许**有空格。
+
+## 配置文件结构
+
+在 `config/_base_` 文件夹下有4个基本组件类型，分别是 数据集(dataset)，模型(model)，训练策略(schedule)和运行时的默认设置(default runtime)。许多方法，例如 Faster R-CNN、Mask R-CNN、Cascade R-CNN、RPN、SSD 能够很容易地构建出来。被来自 `_base_` 下的组件组成的配置我们称为 _原始配置(primitive)_。
```
Comment (hhaAndroid) [inline docs_zh-CN/tutorials/config.md:23]: (1) 4  (2) 分别是 数据集

File: docs_zh-CN/tutorials/config.md [hunk 3]
```diff
@@ -1 +1,525 @@
...
+                        std=[58.395, 57.12, 57.375],
+                        to_rgb=True),
+                    dict(type='Pad', size_divisor=32),
+                    dict(type='ImageToTensor', keys=['img']),
+                    dict(type='Collect', keys=['img'])
+                ])
+        ],
+        samples_per_gpu=2  # 单个 GPU 测试时的 Batch size
+        ))
+evaluation = dict(  # 构建评估钩 (evaluation hook) 的配置，更多细节请参考 https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/evaluation/eval_hooks.py#L7。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:369]: The translation of `hook`. 挂载方法？挂载部件？挂载函数？or do not translate.
And check other translations of `hook`.

File: docs_zh-CN/tutorials/config.md [hunk 4]
```diff
@@ -1 +1,525 @@
...
+log_level = 'INFO'  # 日志的级别。
+load_from = None  # 从一个给定路径里加载模型作为预训练模型，它并不会消耗训练时间。
+resume_from = None  # 从给定路径里恢复检查点(checkpoints)，训练模式将从检查点保存的轮次开始恢复训练。
+workflow = [('train', 1)]  # runner 的工作流程，[('train', 1)] 表示只有一个工作流且工作流仅执行一次。根据 total_epochs 工作流训练 12个回合。
+work_dir = 'work_dir'  # 用于保存当前实验的模型检查点和日志的目录文件地址。
+```
+
+## 常问问题 (FAQ)
+
+### 忽略基础配置文件里的一些域内容。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:407]: 忽略基础配置文件里的部分内容 or 忽略基础配置文件里的某些域的内容。

File: docs_zh-CN/tutorials/config.md [hunk 5]
```diff
@@ -1 +1,525 @@
...
+                num_blocks=(4, 4, 4, 4),
+                num_channels=(32, 64, 128, 256)))),
+    neck=dict(...))
+```
+
+`_delete_=True` 将用新的键去替换 `backbone` 域内所有老的键。
+
+### 使用配置文件里的中间变量
+
+配置文件里会使用一些中间变量，例如数据集里的 `train_pipeline`/`test_pipeline`。 需要注意的是，在子配置文件里修改中间变量时，使用者需要再次传递这些变量给对应的域。例如，我们想改变在训练或测试时，Mask R-CNN 的多尺度策略 (multi scale strategy)，`train_pipeline`/`test_pipeline` 是我们想要修改的中间变量。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:472]: 1. 例如，我们想在训练或测试时，改变 Mask R-CNN ...

**Review by AronLin — COMMENTED** [review]
Please check the space again. Some comments are just suggestions, you can decide to change or not to change according to your own thought.

**Comment by ZwwWayne** [comment]
kindly ping @BIGWangYuDong

**Comment by BIGWangYuDong** [comment]
Kindly ping @AronLin

File: docs_zh-CN/tutorials/config.md [hunk 6]
```diff
@@ -1 +1,524 @@
-# 教程 1: 配置文件
+# 教程 1: 学习配置文件
+
+我们在配置文件中支持了继承和模块化，这便于进行各种实验。如果需要检查配置文件，可以通过运行 `python tools/misc/print_config.py /PATH/TO/CONFIG` 来查看完整的配置
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:3]: A period is missing at the end.

File: docs_zh-CN/tutorials/config.md [hunk 7]
```diff
@@ -1 +1,524 @@
...
+
+当运行 `tools/train.py` 和 `tools/test.py` 时，可以通过 `--cfg-options` 来修改配置文件。
+
+- 更新字典链中的配置
+
+  可以按照原始配置文件中的 dict 键顺序地指定配置预选项。例如，使用 `--cfg-options model.backbone.norm_eval=False` 将模型主干网络中的所有 BN 模块都改为 `train` 模式。
+
+- 更新配置列表中的键
+
+  在配置文件里，一些字典型的配置被包含在列表中。。例如，数据训练流程 `data.train.pipeline` 通常是一个列表，比如 `[dict(type='LoadImageFromFile'), ...]`。如果需要将 `'LoadImageFromFile'` 改成 `'LoadImageFromWebcam'`，需要写成下述形式： `--cfg-options data.train.pipeline.0.type=LoadImageFromWebcam`。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:15]: There are two periods.

File: docs_zh-CN/tutorials/config.md [hunk 8]
```diff
@@ -1 +1,524 @@
...
+
+- 更新列表或元组的值
+
+  如果要更新的值是列表或元组。例如，配置文件通常设置 `workflow=[('train', 1)]`，如果需要改变这个键，可以通过 `--cfg-options workflow="[(train,1),(val,1)]"` 来重新设置。需要注意，引号 \" 是支持列表或元组数据类型所必需的，并且在指定值的引号内**不允许**有空格。
+
+## 配置文件结构
+
+在 `config/_base_` 文件夹下有 4 个基本组件类型，分别是 数据集(dataset)，模型(model)，训练策略(schedule)和运行时的默认设置(default runtime)。许多方法，例如 Faster R-CNN、Mask R-CNN、Cascade R-CNN、RPN、SSD 能够很容易地构建出来。由 `_base_` 下的组件组成的配置，被我们称为 _原始配置(primitive)_。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:23]: There is an extra space after `分别是`.

File: docs_zh-CN/tutorials/config.md [hunk 9]
```diff
@@ -1 +1,524 @@
...
+# 已经弃用的形式
+model = dict(
+   type=...,
+   ...
+)
+train_cfg=dict(...)
+test_cfg=dict(...)
+```
+
+The migration example is as below.
```
Comment (AronLin) [inline docs_zh-CN/tutorials/config.md:66]: Translate?

**Review by AronLin — COMMENTED** [review]
It can be approved after fixing these typos.

**Review by AronLin — APPROVED** [review]
LGTM

File: docs_zh-CN/tutorials/config.md [hunk 10]
```diff
@@ -1 +1,524 @@
...
+
+在 `config/_base_` 文件夹下有 4 个基本组件类型，分别是：数据集(dataset)，模型(model)，训练策略(schedule)和运行时的默认设置(default runtime)。许多方法，例如 Faster R-CNN、Mask R-CNN、Cascade R-CNN、RPN、SSD 能够很容易地构建出来。由 `_base_` 下的组件组成的配置，被我们称为 _原始配置(primitive)_。
+
+对于同一文件夹下的所有配置，推荐**只有一个**对应的**原始配置**文件。所有其他的配置文件都应该继承自这个**原始配置**文件。这样就能保证配置文件的最大继承深度为 3。
+
+为了便于理解，我们建议贡献者继承现有方法。例如，如果在 Faster R-CNN 的基础上做了一些修改，用户首先可以通过指定 `_base_ = ../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py` 来继承基础的 Faster R-CNN 结构，然后修改配置文件中的必要参数以完成继承。
+
+如果你在构建一个与任何现有方法不共享结构的全新方法，那么可以在 `configs` 文件夹下创建一个新的例如 `xxx_rcnn` 文件夹。更多细节请参考 [MMCV](https://mmcv.readthedocs.io/en/latest/understand_mmcv/config.html) 文档。
+
+## 配置文件名称样式
```
Comment (ZwwWayne) [inline docs_zh-CN/tutorials/config.md:31]: 样式 -> 风格

File: docs_zh-CN/tutorials/config.md [hunk 11]
```diff
@@ -1 +1,524 @@
...
+        frozen_stages=1,
+        norm_cfg=dict(type='BN', requires_grad=True),
+        norm_eval=True,
+        style='pytorch'),
+    neck=dict(...),
+    rpn_head=dict(...),
+    roi_head=dict(...))
+```
+
+`ResNet` 和 `HRNet` 使用不同的关键词去构建。
```
Comment (ZwwWayne) [inline docs_zh-CN/tutorials/config.md:430]: Move line 467 here:
基础配置的 Mask R-CNN 使用 ResNet-50，在需要将主干网络改成 HRNet 的时候，因为 HRNet 和 ResNet 中有不同的字段，需要使用 `_delete_=True` 将新的键去替换 `backbone` 域内所有老的键。

File: docs_zh-CN/tutorials/config.md [hunk 12]
```diff
@@ -1 +1,524 @@
...
+                num_channels=(32, 64, 128, 256)))),
+    neck=dict(...))
+```
+
+`_delete_=True` 将用新的键去替换 `backbone` 域内所有老的键。
+
+### 使用配置文件里的中间变量
+
+配置文件里会使用一些中间变量，例如数据集里的 `train_pipeline`/`test_pipeline`。 需要注意的是，在子配置文件里修改中间变量时，使用者需要再次传递这些变量给对应的域。例如，我们想在训练或测试时，改变 Mask R-CNN 的多尺度策略 (multi scale strategy)，`train_pipeline`/`test_pipeline` 是我们想要修改的中间变量。
+
```
Comment (ZwwWayne) [inline docs_zh-CN/tutorials/config.md:472]: 我们在定义新的 `train_pipeline`/`test_pipeline` 之后，需要将它们传递到 `data` 里。

File: docs_zh-CN/tutorials/config.md [hunk 13]
```diff
@@ -1 +1,524 @@
...
+]
+data = dict(
+    train=dict(pipeline=train_pipeline),
+    val=dict(pipeline=test_pipeline),
+    test=dict(pipeline=test_pipeline))
+```
+
+我们首先定义新的 `train_pipeline`/`test_pipeline` 然后传递到 `data` 里。
+
+同样的，如果我们想从 `SyncBN` 切换到 `BN` 或者 `MMSyncBN`，我们需要配置文件里的每一个 `norm_cfg`。
```
Comment (ZwwWayne) [inline docs_zh-CN/tutorials/config.md:515]: 需要修改配置文件里的每一个
